src/main/java/se/sundsvall/businessrules/rule/impl/parkingpermit/criteria/ExpiringParkingPermitCriteria.java

```diff
@@ -38,7 +38,7 @@ public class ExpiringParkingPermitCriteria implements Criteria {
 	private static final int EXPIRATION_PERIOD_IN_MONTHS = 2;
 
 	// CitizenAssets parameter constants.
-	private static final String TYPE = "PERMIT";
+	private static final String TYPE = "PARKINGPERMIT";
```

Comment:
Instead of repitition, what do you think of creating a CitizenAssetsConstants class in the package: se.sundsvall.businessrules.integration.citizenasset. Then you can move all this into this class instead?